micro-deps/build.gradle

```diff
@@ -22,6 +22,31 @@ project(':micro-deps-root:micro-deps') {
     }
 }
 
+project(':micro-deps-root:micro-deps-jax-rs') {
```

Comment:
Spring Boot is using solution 1, example:
spring-boot-starter-undertow
spring-boot-starter-tomcat
spring-boot-starter-jetty

And it's not hystrix it's feign